xrpl/core/binarycodec/definitions/definitions.py

```diff
@@ -59,6 +59,29 @@ def load_definitions(filename: str = "definitions.json") -> Dict[str, Any]:
     value: key for (key, value) in _DEFINITIONS["LEDGER_ENTRY_TYPES"].items()
 }
 
+_GRANULAR_PERMISSIONS = {
+    "TrustlineAuthorize": 65537,
+    "TrustlineFreeze": 65538,
+    "TrustlineUnfreeze": 65539,
+    "AccountDomainSet": 65540,
+    "AccountEmailHashSet": 65541,
+    "AccountMessageKeySet": 65542,
+    "AccountTransferRateSet": 65543,
+    "AccountTickSizeSet": 65544,
+    "PaymentMint": 65545,
+    "PaymentBurn": 65546,
+    "MPTokenIssuanceLock": 65547,
+    "MPTokenIssuanceUnlock": 65548,
+}
+_DELEGATABLE_PERMISSIONS_STR_TO_CODE_MAP: Dict[str, int] = {
+    **{key: value + 1 for (key, value) in _DEFINITIONS["TRANSACTION_TYPES"].items()},
+    **_GRANULAR_PERMISSIONS,
+}
+_DELEGATABLE_PERMISSIONS_CODE_TO_STR_MAP: Dict[int, str] = {
+    **{value + 1: key for (key, value) in _DEFINITIONS["TRANSACTION_TYPES"].items()},
+    **{value: key for (key, value) in _GRANULAR_PERMISSIONS.items()},
+}
```

Comment:
nit (I think this is slightly better but it's fine if you don't want to change this):
```suggestion
_tx_delegations = {key: value + 1 for (key, value) in _DEFINITIONS["TRANSACTION_TYPES"].items()}
_DELEGATABLE_PERMISSIONS_STR_TO_CODE_MAP: Dict[str, int] = {
    **_tx_delegations,
    **_GRANULAR_PERMISSIONS,
}
_DELEGATABLE_PERMISSIONS_CODE_TO_STR_MAP: Dict[int, str] = {
    **{value: key for (key, value) in _DELEGATABLE_PERMISSIONS_STR_TO_CODE_MAP.items()}
}
```